doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,72 @@
+Electron Holography
+*******************
+
+HyperSpy provides the user with two classes which can be used to process electron microscopy data:
+ 

```

Comment:
"which can be used to process electron holography data" would probably be better, as EDS/EELS are still electron microscopy data.